getting-started/index.rst

```diff
@@ -1,31 +1,126 @@
 .. _OPEA_GSG:
 
-Getting Started
-###############
+Getting Started with OPEA
+#########################
 
-1. Try with AI Catalog
+The OPEA ecosystem consists of several complete :ref:`GenAI Examples <GenAIExamples>`
+put together using building-block like :ref:`GenAI microservices <GenAIMicroservices>`.
+You can run these examples as provided, and then customize them to your own
+implementation's needs.or You can also build your own solutions from scratch
+using these microservices.
 
-#. ChatQnA summary
+See these and other Powered by OPEA solutions in action on the
+`AI Explorer`_ site, hosted by Intel:
 
-   a. Default LLM used, additional LLM supported
+.. image:: images/AI-Explorer.png
+   :width: 600px
+   :align: center
+   :class: drop-shadow
+   :target: https://aiexplorer.intel.com/explore
 
-      1. How modify LLMs
+.. _AI Explorer: https://aiexplorer.intel.com/explore
 
-   #. Knowledge base
+-----
 
-      1. Default RAG knowledge base
-      #. BKMs for knowledge base customization
-      #. How to use your own knowledge base
+Get Started
+***********
 
-#. Deployment options
+Prerequisites
+=============
 
-   a. Cloud
-   #. AI enabled PC
+* Python
+* Docker
+* HuggingFace API Token
+* Prometheus and Grafana (optional, needed for evaluation purposes)
 
-      1. Supported platforms
-      #. Deployment
-      #. ChatQnA customization.
+1. **Determine where you will run your solution.**
 
-   #. Complete Deployment Options (links to Deployment section)
+   a. Basic deployment options suitable for proof of concept includes single architectures such as:
+
+      * An AI enable PC.
+      * A server with Intel Xeon Processors.
+
+   b. Production ready environments include:
+
+      * Onprem or cloud based environments (including environments with NVIDIA GPUs).
```

Comment:
s/based environments /based cluster environments running Kubernetes as the orchestrator.

Production environments are typically multi-node to handle scale and resiliency.
We have only tested Kubernetes environments, the flavor of Kubernetes does not matter. Currently at version 1.30